src/libraries/System.Collections.Specialized/tests/NameValueCollection/NameValueCollection.CtorTests.cs

```diff
@@ -152,11 +176,14 @@ public static IEnumerable<object[]> Ctor_Int_IEqualityComparer_TestData()
             yield return new object[] { 0, new IdiotComparer() };
             yield return new object[] { 10, new IdiotComparer() };
             yield return new object[] { 1000, new IdiotComparer() };
-            yield return new object[] { 0, null };
-            yield return new object[] { 10, null };
+            if (PlatformDetection.IsNotHybridGlobalizationOnBrowser)
+            {
+                yield return new object[] { 0, null };
```

Comment:
why this doesn't work ?

Reply:
Because comparer is null, so it's the default one. We don't have any means of checking if the run is HybridGlobalization or not in System.Collections.Specialized, so all the responsibility lies on the user to use the correct constructor. Here in the test the incorrect one is used.